### PR TITLE
Fixes 179

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by Kraseo** [comment]
>fixes 179
>commit log
>fixes 169

**Comment by Cyberboss** [comment]
Shush

**Comment by coiax** [comment]
Oh I thought it was like a dynamic time calculation based on previous startups. My world is ruined. :(

File: code/modules/mob/new_player/login.dm [hunk 1]
```diff
@@ -29,4 +29,4 @@
 	new_player_panel()
 	client.playtitlemusic()
 	if(ticker.current_state < GAME_STATE_SETTING_UP)
-		src << "Please set up your character and select \"Ready\". The game will start in about [round(ticker.GetTimeLeft()/10)] seconds."
\ No newline at end of file
+		src << "Please set up your character and select \"Ready\". The game will start in about [round((ticker.GetTimeLeft() + 1)/10)] seconds."
```
Comment (MrStonedOne) [inline code/modules/mob/new_player/login.dm]: the real answer here is that round() is a floor() if you don't give it a second arg, so just make it a real round by doing round(value, 1)

**Pull request merged**